### ISSUE TITLE
Export to Riverscapes Project Not working

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/QRiS#308]
Using either of @zburgert1997 demo QRiS projects for Willow Springs (https://drive.google.com/open?id=1OYncRBqudw0C761lsm6qqGALrXZWDTBZ&usp=drive_fs) & Bailey Flat (https://drive.google.com/open?id=1OaisNXsaFw43Kg0bVvu41tplq_Vs1vvj&usp=drive_fs). 

Export just crashes in various ways. 

```
An error has occurred while executing Python code: 

ValueError: Null geometry cannot be converted to a point. 
Traceback (most recent call last):
  File "C:\Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qris_deploy\src\view\frm_export_project.py", line 290, in accept
    centroid = envelope.centroid().asPoint()
ValueError: Null geometry cannot be converted to a point.


Python version: 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] 
QGIS version: 3.26.3-Buenos Aires Buenos Aires, 65e4edfdada 

Python Path:
C:/PROGRA~1/QGIS32~2.3/apps/qgis/./python
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS32~2.3/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.26.3\bin\python39.zip
C:\PROGRA~1\QGIS32~2.3\apps\Python39\DLLs
C:\PROGRA~1\QGIS32~2.3\apps\Python39\lib
C:\Program Files\QGIS 3.26.3\bin
C:\Users\joemw\AppData\Roaming\Python\Python39\site-packages
C:\PROGRA~1\QGIS32~2.3\apps\Python39
C:\PROGRA~1\QGIS32~2.3\apps\Python39\lib\site-packages
C:\PROGRA~1\QGIS32~2.3\apps\Python39\lib\site-packages\win32
C:\PROGRA~1\QGIS32~2.3\apps\Python39\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS32~2.3\apps\Python39\lib\site-packages\Pythonwin
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:\Users\joemw\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\mapflow\dialogs\widgets
C:\Users\joemw\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\mmqgis/forms
C:\Users\joemw\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qris_deploy\src\..\wheels\rsxml-2.0.1-py3-none-any.whl
```

**Comment by Jdgilby** [comment]
@joewheaton it might be the project. They were corrupted for some reason after I zipped them up. I took that same project that kelly fixed and got a successful export....However @KellyMWhitehead I got a very familiar error when trying to open the project in the viewer.
![image](https://github.com/Riverscapes/QRiS/assets/97305413/6f220aa1-0ab3-4699-976c-d6d5a7305ad1)

@joewheaton try exporting this qris project. 
[Willow_Springs_036.zip](https://github.com/Riverscapes/QRiS/files/13845933/Willow_Springs_036.zip)

**Comment by joewheaton** [comment]
If we have any projects that successfully exported @KellyMWhitehead or @Jdgilby, we need to ASAP do the following:
- [ ] Test opening them in Q Viewer.
- [ ] If Business Logic needs fixing, fix it.
- [ ] Upload to Data Exchange at least one, preferably more projects
- [ ] Curate the QRiS project for web too! 
This is really important functionality to test and implement very soon. @philipbaileynar thoughts?

Beyond that, these bugs on exporting and opening need to be explored and fixed.

**Comment by Jdgilby** [comment]
@KellyMWhitehead Heres the latest attempt
![image](https://github.com/Riverscapes/QRiS/assets/97305413/a744eb26-b7a1-47c3-8deb-26e1989f1420)

**Comment by KellyMWhitehead** [comment]
@Jdgilby Is this the project from above or a newer one?

**Comment by Jdgilby** [comment]
@KellyMWhitehead Oh ya I didn't see that project in the thread. Heres the one that isnt working. https://drive.google.com/file/d/1kJgXt0NNRcTeegIW1XfKhu2vTLUOV7BU/view?usp=drive_link

**Comment by KellyMWhitehead** [comment]
@Jdgilby the provided link contains an empty zip file...

**Comment by Jdgilby** [comment]
@KellyMWhitehead Sorry, try it now. It wasnt fully uploaded until just now.

**Comment by KellyMWhitehead** [comment]
Fixed for version 0.3.22

**Comment by Jdgilby** [comment]
@KellyMWhitehead Everything looks good except it appears that Sample Frames were not included in project.rs.xml.
![image](https://github.com/Riverscapes/QRiS/assets/97305413/d85bd557-e306-4c84-9641-2ca3aed3defb)

**Comment by KellyMWhitehead** [comment]
Fixed for 0.3.23.
